modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/definition/tree/manager/test/ObjectDefinitionTreeUtilTest.java

```diff
@@ -1441,9 +1510,11 @@ public void testUpdateNodeObjectDefinition() throws Exception {
 	}
 
 	@Test
-	public void testUpdateNodeObjectDefinitionWithDraftNodeDescendants()
+	public void testUpdateNodeObjectDefinitionWithNodeDescendants()
```

Comment:
I believe it should be: testUpdateNodeObjectDefinitionWith**DescendantNodes**

it follows the method name and the comments in this test.